### ISSUE TITLE
Export

### ISSUE THREAD

**Issue by dc500** [jmfernandes/robin_stocks#167]
The export (not download docs) feature not working for me. The last time I was able to use it was on the 9th. (3 days ago). Is anyone else having problems?  *.get[.......] does not work at all. It now returns nothing. FYI Yesterday, I got the infamous email from RH. Could that have something to do with it?

**Comment by oracle3987** [comment]
Why are you in a place you dont belong . You will be in violation and monitored

**Comment by unrealgecko** [comment]
what is the infamous email?

**Comment by dc500** [comment]
> what is the infamous email?

It's an email stating that RH users are not to use the api to TRADE. The app prompts the user to upload their ID and reply to the email.

**Comment by dc500** [comment]
> Why are you in a place you dont belong . You will be in violation and monitored 


As soon as my options expire, I'm never looking back. RH is the wrong platform for millennial day traders. $20k loss due to their glitches (clearly they don't understand routing), their quotes are beyond off (I am forced to watch my ticker on WeBull. Doing so allows me to profit with every trade - real time) and limited trade hours. I lose over $1200 per day because PM and AH are limited. So you're right, I have no business using RH. It's funny. After I got the email, I didn't use robin stocks again but I found a way to manually do more trades than I was doing with the script. If I get banned it's because I found a [manual] way to outsmart and perform. Since working with RS, my strategy (and confidence) has improved greatly. I guess it's term to convert my Alpaca to a DT account. I wrote my script last night. Can't wait until my funds clear. 👍
